static/js/zamboni/collections.js

```diff
@@ -268,6 +268,8 @@ if (addon_ac.length) {
                 .data('item.autocomplete', item)
                 .append('<a><img src="' + item.icons['32'] + '" alt="">' + _.escape(item.name) + '</a>')
                 .appendTo(ul);
+        } else {
+            return ul;
```

Comment:
Ah, I thought `.appendTo(ul)` was returning the `ul`. I'm changing this line to `return $('<li>')`.

This current code works well locally though, which is odd.